src/main/java/bio/terra/service/snapshot/flight/delete/DeleteSnapshotDependencyDataAzureStep.java

```diff
@@ -57,8 +58,12 @@ public StepResult doStep(FlightContext context) {
             datasetBillingProfile, datasetStorageAccountResource);
     TableServiceClient datasetTableServiceClient =
         azureAuthService.getTableServiceClient(datasetAzureStorageAuthInfo);
-    tableDependencyDao.deleteSnapshotFileDependencies(
-        datasetTableServiceClient, dataset.getId(), snapshotId);
+    try {
+      tableDependencyDao.deleteSnapshotFileDependencies(
```

Comment:
This is the step that's failing in the scale test clean-up

Reply:
I wonder what happens if we add batching?  What kind of failure are you getting?

Reply:
There's a failure when we try to delete one of the entities:
```
Step Result exception com.azure.data.tables.models.TableTransactionFailedException: An action within the operation failed, the transaction has been rolled back. The failed operation was: DeleteEntity{partitionKey='2f23518d-af19-465f-9f7d-09fcbc380418', rowKey='8e4e6a35-cc98-4697-bda4-e6efac507f28', eTag='W/"datetime'2021-12-10T16%3A07%3A56.2935684Z'"', ifUnchanged=false}```

I'll see if batching helps!